src/components/toolbar/README.md

```diff
@@ -0,0 +1,59 @@
+# MdToolbar
+`MdToolbar` is a vertical aligned bar, which can hold the application title or actions.
+
+### Screenshots
+![Preview](https://cloud.githubusercontent.com/assets/4987015/13727769/6d952c78-e900-11e5-890a-ccfd46996812.png)
+
+## `<md-toolbar>`
+### Bound Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `color` | `"primary" | "accent" | "warn"` | The color palette for the toolbar |
+
+### Notes
+By the default the `md-toolbar` will use the background palette.
+
+Multiple rows inside of a `md-toolbar` can be added by appending as many as needed `<md-toolbar-row>` elements.
+
+### Example
+```html
+<md-toolbar [color]="myColor">
+  <span>First Toolbar Row</span>
+
+  <md-toolbar-row>
+    <span>Second Toolbar Row</span>
+  </md-toolbar-row>
+  
+  <md-toolbar-row>
+    <span>Third Toolbar Row</span>
+  </md-toolbar-row>
+</md-toolbar>
+```
+
+### Alignment
+The `md-toolbar` component doesn't provide any API to align the items probably in the row.
+
+The alingment inside of a toolbar row can be easily done by using flexbox properties.<br/>
+For example, the following markup aligns the items on the `right`.
+
+Custom HTML
+```html
+<md-toolbar>
+  <span>Application Title</span>
+  
+  <!-- This fills the remaining space of the current row -->
+  <span class="demo-fill"></span>
+  
+  <span>Right Alingned Text</span>
+</md-toolbar>
+```
```

Comment:
It would be good to add a screenshot immediately after this example to show the result.